assets/scripts/menus/SignInMenu.jsx

```diff
@@ -30,12 +45,7 @@ export class SignInMenu extends React.Component {
           </svg>
           <FormattedMessage id="menu.signin.twitter" defaultMessage="Sign in with Twitter" />
         </a>
-        <a href="#" onClick={this.onClickEmailSignIn}>
-          <svg className="icon">
-            <use xlinkHref="#icon-forums" />
-          </svg>
-          <FormattedMessage id="menu.signin.email" defaultMessage="Sign in with Email" />
-        </a>
+        {this.renderEmailAuthLink()}
```

Comment:
We've started to gravitate toward a much simpler way of doing this:

```js
{this.props.emailAuthEnabled && (
   <your html>
       ...
   </your html>
)}
```

Easier to clean up if the flag goes away!